src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/CancelLink.jsx

```diff
@@ -13,6 +15,14 @@ function formatAppointmentDate(date) {
 
 export default function CancelLink({ appointment }) {
   const dispatch = useDispatch();
+  const showCancelButton = useSelector(selectFeatureCancel);
+  const canceled = appointment.status === APPOINTMENT_STATUS.cancelled;
+  const isPastAppointment = appointment.vaos.isPastAppointment;
+
+  if (canceled || !showCancelButton || isPastAppointment) {
```

Comment:
I think this would read easier if we combined these conditionals into a single variable. I'm finding that a lot of our more difficult to read components use this multiple condition render logic.

Same with the conditional render in the `CalendarLink` component.

Reply:
Something like this?

```js
const hideForCanceledAndPast = canceled || !showCancelButton || isPastAppointment;

if (hideForCanceledAndPast) {
  return null;
}
```

Reply:
Yeah, exactly what I was thinking.